docs/AI-for-security/attack-discovery.asciidoc

```diff
@@ -33,6 +33,14 @@ This page describes:
 * <<attack-discovery-what-info, What information each discovery includes>>
 * <<attack-discovery-workflows, How you can interact with discoveries to enhance {elastic-sec} workflows>>
 
+[discrete]
+[[attack-discovery-rbac]]
+== Role-based access control (RBAC) for Attack Discovery
```

Comment:
There's a bit of inconsistency with how attack discovery is capitalized throughout the page (sentence case vs title case). Not sure which one it should be.